astropy/coordinates/sky_coordinate_parsers.py

```diff
@@ -568,7 +555,7 @@ def _parse_coordinate_arg(coords, frame, units, init_kwargs):
                     "Input coordinate values must have same number of elements, found"
                     f" {n_coords}"
```

Comment:
To keep the exception message the same, maybe do `{sorted(n_coords)}` here.

Reply:
Good suggestion.